docs/communication/communication.md

```diff
@@ -11,23 +11,28 @@ asynchronous communication as starting point.
 We place emphasis on documentation and ensure that decisions and conclusions
 are written down and accessible to everyone in the company.
 
+## Guidelines
+
+1. All announcements to Thoth Tech general channel only _(clarity in notifications)_
+1. Use channels for all general to major discussion
+1. Use chat / direct message for private or personal discussion
+1. Use GitHub PRs for feedback on code or documentation _(building visibility)_
+1. Use Trello for planning and assigning tasks
+1. Use meetings for sychronous discussion _(for private channels, record and store the meeting in the channel files)_
+1. Use whiteboard tools (Miro or MS) to collaborate in meetings _(send a link in the channel to the board)_
+
 ## Tips and practices
 
-1. You are not expected to be available all the time. There is no expectation
-   to respond to messages outside of your planned working hours.
+1. You are not expected to be available all the time. There is no expectation to respond to messages outside of your planned working hours.
 1. All company data should be shareable by default.
-1. It is OK to ask as many questions as you have. Please do them in a way so
-   many people can answer them and many people see the answer. Use Pull Requests
-   or public chat channels instead of direct messages.
+1. It is OK to ask as many questions as you have. Please do them in a way so many people can answer them and many people see the answer. Use Pull Requests or public chat channels instead of direct messages.
```

Comment:
**suggestion:** The maximum line length is set to 80 (it is set in the prettier [config](https://github.com/thoth-tech/handbook/blob/main/.prettierrc). We set a short line length to allow better readability, for example, preview split-screen or in code review. Could you kindly merge the changes from master `git merge master` to get the required config 🙏🏼 ?

Reply:
I am pretty sure I did this correctly, could you please double check as part of the review. Thank you